admin-api-server/test/app.e2e.spec.ts

```diff
@@ -2007,6 +2007,8 @@ describe('AppController (e2e)', () => {
             currency: 'EUR',
             edition_size: 1,
             transaction_value: '0.82',
+            fee_in_base_currency: null,
+            purchaser_country: null,
```

Comment:
the `emulateNftSale` test utility function should be amended such that it fakes the purchaser country to something (and make it a parameter to the `emulateNftSale` function)

Reply:
added the purchaser_country as an optional param to emulateNftSale that defaults to 'GB'